meinberlin/react/account/NotificationsList.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from 'react'
+import django from 'django'
+import { ToggleSwitch } from '../contrib/ToggleSwitch'
+import NotificationToggle from './NotificationToggle'
+
+const notifications = [
+  {
+    header: django.gettext('Project-Related Notifications'),
+    notifications: {
+      email_newsletter: {
+        title: django.gettext('E-Mail Newsletter'),
+        description: django.gettext('Receive newsletters with updates and news about the projects you follow via e-mail.'),
+        showInActivityFeed: true
```

Comment:
In the design the email newsletter has no `Show in activity feed` button which kind of makes sense for an email I think ? :)

Reply:
ah good catch, thanks D: